modules/machineset-yaml-alibaba.adoc

```diff
@@ -136,6 +136,11 @@ endif::infra[]
 <12> Specify the zone within your region to place machines on. Be sure that your region supports the zone that you specify.
 ifdef::infra[]
 <13> Specify a taint to prevent user workloads from being scheduled on infra nodes.
++
+[NOTE]
+====
+After adding the `NoSchedule` taint on the infrastructure node, existing pods running on that node are marked as `misscheduled`. You must either delete `misscheduled` pods or link:https://access.redhat.com/solutions/6592171[add toleration on `misscheduled` pods].
```

Comment:
i think it will dns pod on the node will be marked misscheduled not all pods

Reply:
@melvinjoseph86 Addressed- can you please take a look now?

Reply:
looks better, thanks